File: app/src/main/java/com/baozi/mvp/view/PagerFragmentView.java
package com.baozi.mvp.view;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.view.ViewPager;
import android.view.View;

import java.util.List;

/**
 * Created by Administrator on 2017/8/8 0008.
 */

public interface PagerFragmentView extends PagerView {

    FragmentManager getFgManager();

    Class<? extends Fragment>[] getFragments();
}

File: lib-mvpbase-kotlin/src/main/java/com/linfeng/mvp/view/PagerFragmentView.kt
package com.linfeng.mvp.view

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.view.ViewPager

/**
 * Created by Administrator on 2017/8/8 0008.
 */

interface PagerFragmentView : PagerView {

    override val viewPager: ViewPager

    val fgManager: FragmentManager

    val fragments: Array<Class<Fragment>>

}
